src/delay.rs

```diff
@@ -54,17 +55,22 @@ impl Delay {
 
         self.syst.disable_counter();
     }
+
+    /// Delay using the Cortex-M systick for a certain duration, ms.
+    pub fn delay_ms(&mut self, ms: u32) {
+        self.delay_us(ms * 1_000);
```

Comment:
This implementation is correct not for all argument values. Please see the `delay_ms` implementation below.